src/date/expand-pattern.js

```diff
@@ -33,10 +55,30 @@ return function( pattern, cldr ) {
 
 	switch ( true ) {
 		case "skeleton" in pattern:
+			skeleton = pattern.skeleton;
 			result = cldr.main([
 				"dates/calendars/gregorian/dateTimeFormats/availableFormats",
-				pattern.skeleton
+				skeleton
 			]);
+			if ( result === undefined ) {
+				for ( i = 0 ; i < skeleton.length ; i++ ) {
+					if ( /[hHms]/.test( skeleton[i] ) ) {
+						timeSkeleton += skeleton[i];
+					} else if ( /[GyQMEd]/.test( skeleton[i] ) ) {
+						dateSkeleton += skeleton[i];
+					}
```

Comment:
Why `hHms` and `HyQMEd` only? You have ignored what I've said in https://github.com/jquery/globalize/pull/426#issuecomment-85226376 "allowed patterns of a skeleton are `GyYuUrQqMLlwWdDFgEechHKkmsSAzZOvVXx`"